src/Helper/Navigation/PluginManager.php

```diff
@@ -81,6 +48,7 @@ public function __construct($configOrContainerInstance = null, array $v3config =
             $instance->setServiceLocator($this->creationContext);
         };
 
-        parent::__construct($configOrContainerInstance, $v3config);
+        $this->aliases   = ConfigProvider::defaultViewHelperAliases();
```

Comment:
by adding this "after" the `aliases`, you might override additional aliases which were passed via `config` construct argument.

so you should either use `array_replace_recursive($defaults, $configFromConstruct)` or set the properties before actually calling `__construct`.

A unit test to verify that everything works as expected would be very appreciated.
especially for this kind of refactoring.